src/test/java/org/openstreetmap/atlas/utilities/scalars/AngleTest.java

```diff
@@ -153,6 +153,16 @@ public void testPositiveRadians()
         Assert.assertEquals(Math.PI, value.asPositiveRadians(), 1e-9);
     }
 
+    @Test
+    public void testReverse()
+    {
+        Assert.assertEquals(Angle.degrees(0), Angle.MINIMUM.reverse());
+        Assert.assertEquals(Angle.degrees((double) -1 / Angle.DM7_PER_DEGREE),
+                Angle.MAXIMUM.reverse());
+        Assert.assertEquals(Angle.degrees(90), Angle.degrees(-90).reverse());
+        Assert.assertEquals(Angle.degrees(-90), Angle.degrees(90).reverse());
```

Comment:
Can you also add test cases where there is some modulo happening? Like
`Assert.assertEquals(Angle.degrees(90), Angle.degrees(-450).reverse());`